.all-contributorsrc

```diff
@@ -0,0 +1,24 @@
+{
+  "projectName": "user-data-for-fraud-prevention",
+  "projectOwner": "skodamarthi",
```

Comment:
Is it possible to add two project owners? Can you please add @reubenae also as a project owner?

Reply:
I checked the docs and it seems like the value cannot be an array, so I can only have 1 project owner: https://allcontributors.org/docs/en/cli/configuration. :(

Reply:
I am okay without the credit here :) I think I'm listed in the package.json so its all good